script.js

```diff
@@ -317,12 +325,72 @@ const treeTable = {
     }
 }
 
+const categoriesTable = {
+    rows: [
+        {
+            view: "toolbar",
+            elements: [
+                {
+                    view: "text",
+                    id: "category_input",
+                    placeholder: "Category name",
+                },
+                {
+                    view: "button",
+                    label: "Add Category",
+                    id: "btn_add_category",
```

Comment:
don't set widget id if you do not plan to use it (same for delete button)

Reply:
Thank you, will keep it in mind.